config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  get '/', to: 'tasks#index', as: 'home'
```

Comment:
We could probably also write this explicitly as the root route:
```ruby
root 'tasks#index'
```
and then use `root_path` everywhere instead of `home_path`.